components/gameboard/DiscardPile.tsx

```diff
@@ -1,13 +1,14 @@
 import styles from "./Piles.module.css";
-import { PileProps } from "./types";
+import { DiscardPileProps } from "./types";
```

Comment:
There is a small optimization if you only import types:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

```suggestion
import type { DiscardPileProps } from "./types";
```

Reply:
Thanks, didn't know that.